packages/assert/index.js

```diff
@@ -42,32 +42,18 @@ if (missing.length > 0) {
   );
 }
 
-const {
-  bare,
-  details,
-  equal,
-  error,
-  Fail,
-  note,
-  quote,
-  makeAssert,
-  string: assertString,
-  typeof: assertTypeof,
-} = globalAssert;
+const { bare, details, error, Fail, note, quote, makeAssert } = globalAssert;
```

Comment:
I thought part of the consensus was to have the `assert` exported by this package contain only the assertions, i.e., the methods not separately exported by name.

```suggestion
const { bare, details, error, Fail, note, quote, makeAssert, ...assertions } = globalAssert;

Object.freeze(assertions);

export { assertions as assert };
```

Reply:
Thanks, I forgot to include that change.

Reply:
And the exported `assert` needs to be callable as well.

```js
const { bare, details: redacted, error, Fail: throwRedacted, note, quote,  makeAssert, ...assertions } = globalAssert;
const assert = (...args) => globalAssert(...args);
Object.assign(assert, assertions);
export { assert, redacted, throwRedacted, quote, bare, note, makeAssert };
```